knative-operator/deploy/resources/kourier/kourier.yaml

```diff
@@ -0,0 +1,289 @@
+#apiVersion: v1
+#kind: Namespace
+#metadata:
+#  name: kourier-system
+#---
+apiVersion: networking.k8s.io/v1
```

Comment:
Shall we move this to a different file maybe to signify that this is "from us"?

Reply:
Sure. Done.

Reply:
Done